sources/packages/forms/src/form-definitions/studentadditionaltransportationappeal.json

```diff
@@ -0,0 +1,1124 @@
+{
+  "title": "Appeals - Student Additional Transportation",
+  "name": "studentAdditionalTransportationAppeal",
+  "path": "studentadditionaltransportationappeal",
+  "type": "form",
+  "display": "form",
+  "tags": [
+    "common"
+  ],
+  "components": [
+    {
+      "key": "financialInformation",
```

Comment:
This is business question and I saw that story cares only about the sub question. 

But in other appeals like dependants or financial information, root question was replicated as is from student application. Why is this one different??

Reply:
I'm not sure if I understood but I'm using `eligibleForAnAdditionalTransportationAllowance` now.